packages/shared/src/styles/utilities.css

```diff
@@ -16,6 +16,10 @@
   text-overflow: ellipsis;
 }
 
+.break-word {
```

Comment:
This is already provided by tailwind I think. @rebelchris isn't it?

Reply:
There are some https://tailwindcss.com/docs/word-break
Just by manually applying the CSS, the only one that works is `break-all` class

Reply:
huh, wait a sec. I'll check if my application is up to date

Reply:
Ah, latest release is 3.8.1, and #656 has been merged after latest release

Reply:
@idoshamun perhaps doing a release will fix it in the extension, since it has not gotten the fix from #656 yet

Reply:
The tailwind version is `break-words` but it adds it as:

```css
.break-words {
    overflow-wrap: break-word;
}
```

That's why we add it as:

```css
.title {
  -webkit-line-clamp: 3;
  word-break: break-word;
}
```

The issue at hand is that we never build the Extension after this fix.
So this is only available on the webapp for now.

@omBratteng Would you mind checking it works on the webapp?
https://app.daily.dev/

Reply:
@rebelchris I just checked in the webapp, and it is fixed there. So it is as you said, because there hasn't been a new build of the extension after this fix.
I looked for the changes done in #656 but I could not see them, so I assumed that they didn't work. But I forgot to check the version of the extension and if it worked in the web app.

Reply:
No worries thanks for the keen eye! 👀

Reply:
We will issue a new release soon. Just waiting to finish some massive feature before we do that :)

Reply:
Just a followup, noticed extension was at `v3.9.0`, and I now see the `word-break: break-word` style applied to the title :)

Reply:
@idoshamun @rebelchris Just a followup, noticed extension was at `v3.9.0`, and I now see the `word-break: break-word` style applied to the title 😄